tq/transfer_queue.go

```diff
@@ -633,7 +634,12 @@ func (q *TransferQueue) run() {
 
 // canRetry returns whether or not the given error "err" is retriable.
 func (q *TransferQueue) canRetry(err error) bool {
-	return errors.IsRetriableError(err)
+	switch cause := errors.Cause(err).(type) {
```

Comment:
What would you think about just putting that code in [`errors.IsRetriableError`](https://github.com/git-lfs/git-lfs/blob/b7b1f2b7985c8f7a18c563c70898f669ffb8c13c/errors/types.go#L129-L141) so that all callers recognize `*url.Error`?